pywr-core/src/parameters/mod.rs

```diff
@@ -763,19 +763,19 @@ pub struct ParameterCollectionSize {
 #[derive(Default)]
 pub struct ParameterCollection {
     constant_f64: Vec<Box<dyn ConstParameter<f64>>>,
-    constant_usize: Vec<Box<dyn ConstParameter<usize>>>,
+    constant_usize: Vec<Box<dyn ConstParameter<u64>>>,
```

Comment:
should the attributes be renamed to  `constant_u64`, `simple_u64`, and `simple_u64`?